apps/web/components/v2/apps/App.tsx

```diff
@@ -0,0 +1,337 @@
+                        };
+                      }
+                      return (
+                        <Button
+                          StartIcon={Icon.FiPlus}
+                          {...props}
+                          // @TODO: Overriding color and size prevent us from
+                          // having to duplicate InstallAppButton for now.
+                          color="primary"
+                          size="base"
+                          data-testid="install-app-button">
+                          {t("add_another")}
+                        </Button>
+                      );
+                    }}
+                  />
+                )}
+              </div>
+            ) : installedAppCount > 0 ? (
+              <Button color="secondary" disabled title="App already installed">
+                {t("installed")}
+              </Button>
+            ) : (
+              <InstallAppButton
+                type={type}
+                isProOnly={isProOnly}
+                render={({ useDefaultComponent, ...props }) => {
+                  if (useDefaultComponent) {
+                    props = {
+                      onClick: () => {
+                        mutation.mutate({ type });
+                      },
+                      loading: mutation.isLoading,
+                    };
+                  }
+                  return (
+                    <Button
+                      data-testid="install-app-button"
+                      {...props}
+                      // @TODO: Overriding color and size prevent us from
+                      // having to duplicate InstallAppButton for now.
+                      color="primary"
+                      size="base">
+                      {t("install_app")}
+                    </Button>
+                  );
+                }}
+              />
+            )
+          ) : (
+            <SkeletonButton width="24" height="10" />
+          )}
+          {price !== 0 && (
+            <span className="block text-right">
+              {feeType === "usage-based" ? commission + "% + " + priceInDollar + "/booking" : priceInDollar}
+              {feeType === "monthly" && "/" + t("month")}
+            </span>
+          )}
+
+          <div className="mt-8 space-x-2">{body}</div>
+          <h4 className="mt-8 font-semibold text-gray-900 ">{t("pricing")}</h4>
+          <span>
+            {price === 0 ? (
+              "Free"
+            ) : (
+              <>
+                {Intl.NumberFormat("en-US", {
+                  style: "currency",
+                  currency: "USD",
+                  useGrouping: false,
+                }).format(price)}
+                {feeType === "monthly" && "/" + t("month")}
+              </>
+            )}
+          </span>
+
+          <h4 className="mt-8 mb-2 font-semibold text-gray-900 ">{t("learn_more")}</h4>
+          <ul className="prose-sm -ml-1 -mr-1 leading-5">
```

Comment:
I think we are missing the "App Store / Single App View v2 (when we have more content)" part, which I'm guessing it will take care of the use case when a lot of information is included in the text side of the page. Would you be willing to expand your already impressive work to accommodate for that use case @JeroenReumkens? Check it out: https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=1413%3A76791

Reply:
Oh interesting! I actually assumed this was more of a design experiment rather than a variant that would really be used, since it wasn't mentioned anywhere in the issue 😄  How do we decide based on the MDX what variant to show? Maybe it would make even more sense to abstract a few things into a separate component, rather than trying to combine this into one template — that would complicate the template by a lot I think. Could definitely give this a go once I know how we want to use this.